proto/src/main/proto/Rss.proto

```diff
@@ -592,6 +592,8 @@ message RssReassignFaultyShuffleServerRequest{
   int32 shuffleId  = 1;
   repeated string partitionIds = 2;
   string faultyShuffleServerId = 3;
+  int32 stageId = 4;
```

Comment:
@zuston I have some concern about this. Will it affect the reuse of exchange, partial tasks execution of the shuffle and so on.

Reply:
From the description, I didn't see any requirements for this PR, but I guess it wants to support different reassginment servers for different stage attempt.

For me, currently there is no need to support above stage attempt